movement/io/load_poses.py

```diff
@@ -21,6 +21,59 @@
 logger = logging.getLogger(__name__)
 
 
+def from_file(
+    file_path: Union[Path, str],
+    source_software: Literal["DeepLabCut", "SLEAP", "LightningPose"],
+    fps: Optional[float] = None,
+) -> xr.Dataset:
+    """Load pose tracking data from a DeepLabCut (DLC), LightningPose (LP) or
+    SLEAP output file into an xarray Dataset.
+
+    Parameters
+    ----------
+    file_path : pathlib.Path or str
+        Path to the file containing predicted poses. The file format must
+        be among those supported by the from_dlc_file(), from_slp_file()
+        or from_lp_file() functions.
+    source_software : "DeepLabCut", "SLEAP" or "LightningPose"
+        The source software of the file.
+    fps : float, optional
+        The number of frames per second in the video. If None (default),
+        the `time` coordinates will be in frame numbers.
+
+    Returns
+    -------
+    xarray.Dataset
+        Dataset containing the pose tracks, confidence scores, and metadata.
+
+    Notes
+    -----
+    Identical to calling any of the functions from_dlc_file(),
+    from_sleap_file() or from_lp_file().
+
+    See Also
+    --------
+    movement.io.load_poses.from_dlc_file : Load pose tracks directly
+    from DeepLabCut files.
+    movement.io.load_poses.from_sleap_file : Load pose tracks directly
+    from SLEAP files.
+    movement.io.load_poses.from_lp_file : Load pose tracks directly
+    from LightningPose files.
+
+    """
+
+    if source_software == "DeepLabCut":
+        return from_dlc_file(file_path, fps)
+    elif source_software == "SLEAP":
+        return from_sleap_file(file_path, fps)
+    elif source_software == "LightningPose":
+        return from_lp_file(file_path, fps)
+    else:
+        raise ValueError(
+            "Unsupported source software: {}".format(source_software)
+        )
```

Comment:
If you use out `log_error` utiliy, it will also log the error before saving it.
```python
raise log_error(
    ValueError,
    f"Unsupported source software: {source_software}".
)
```